pr-data.csv

```diff
@@ -3703,7 +3703,7 @@ https://github.com/Graylog2/graylog2-server,87d63f677759336987fbb21fd9c688235618
 https://github.com/Graylog2/graylog2-server,87d63f677759336987fbb21fd9c688235618e7bb,graylog2-server,org.graylog2.utilities.IPSubnetConverterTest.testDefault,ID,Accepted,https://github.com/Graylog2/graylog2-server/pull/7543,
 https://github.com/Graylog2/graylog2-server,87d63f677759336987fbb21fd9c688235618e7bb,graylog2-server,org.graylog2.utilities.IPSubnetConverterTest.testNormalize,ID,Accepted,https://github.com/Graylog2/graylog2-server/pull/7543,
 https://github.com/haraldk/TwelveMonkeys,29f7547a99da1719aa22e7ea1f86b6b93d2fed55,imageio/imageio-core,com.twelvemonkeys.imageio.color.DiscreteAlphaIndexColorModelTest.testCreateCompatibleRaster,ID,,,
-https://github.com/haraldk/TwelveMonkeys,29f7547a99da1719aa22e7ea1f86b6b93d2fed55,imageio/imageio-core,com.twelvemonkeys.imageio.StandardImageMetadataSupportTest.withTextValuesMap,ID,,,
+https://github.com/haraldk/TwelveMonkeys,29f7547a99da1719aa22e7ea1f86b6b93d2fed55,imageio/imageio-core,com.twelvemonkeys.imageio.StandardImageMetadataSupportTest.withTextValuesMap,ID,InspiredAFix,https://github.com/haraldk/TwelveMonkeys/pull/831,https://github.com/TestingResearchIllinois/idoft/issues/1065
```

Comment:
Please put directly here the info from the issue. Thanks!

Reply:
> Please put directly here the info from the issue. Thanks!

What exactly should I put here? All the description from the issue or any specific thing like CommitID of the developer fix?

Reply:
Put in `Notes` something like `developer fix is in https://github.com/haraldk/TwelveMonkeys/commit/be348543d8a41d4677f83e1c69988221527977a9`. Thanks!

Reply:
If it helps, you can change to `https://github.com/haraldk/TwelveMonkeys/commit/be348543d8a41d4677f83e1c69988221527977a9 is the developer fix`.

Reply:
Even that is failing in my forked repo: https://github.com/prathyushreddylpr/idoft/actions/runs/7067132906/job/19240106122?pr=1

Reply:
Check in TestingResearchIllinois/idoft who and when added that specific `Reason...`.